### ISSUE TITLE
kibana.rb does not adhere to port argument

### ISSUE THREAD

**Issue by jib** [rashidkpc/kibana2#158]
```
$ ruby kibana.rb -h
Usage: kibana [options]
    -p port                          set the port (default is 4567)
    -o addr                          set the host (default is 0.0.0.0)
    -e env                           set the environment (default is development)
    -s server                        specify rack server/handler (default is thin)
    -x                               turn on the mutex lock (default is off)

$ kibana.rb -p 8080
== Sinatra/1.3.3 has taken the stage on 5601 for development with backup from Thin
>> Thin web server (v1.5.0 codename Knife)
>> Maximum connections set to 1024
>> Listening on 127.0.0.1:5601, CTRL+C to stop
```

Running it without a -p argument makes it run on 5601, which is also different than the promised 4567.

This is as of commit:

```
$ git log | head -1
commit 25f31b10a9a8da2019b03e7a4c33440467a9c3af
```

Same problem applies to -o and probably all the other arguments.

**Comment by taidan19** [comment]
Two things are going on here.

1) When you run `ruby kibana.rb -h`, the options that show up are generated by Sinatra, not Kibana.

2) Kibana determines which port to run on by looking at the "KibanaPort" property defined in KibanaConfig.rb (it was on line 16 in my version of the file). The way kibana.rb is written, it always sets the port to this value, and it will override any value you set on the command line.

In other words, the output of `-h` is misleading because it isn't generated by Kibana; for the same reason, it also can't be changed.

Second, if you specify your port in KibanaConfig.rb, it should work as you expect.
